lang/semgrep-grammars/src/semgrep-rust/test/corpus/semgrep.txt

```diff
@@ -91,6 +91,57 @@ impl Foo {
       (function_item (identifier) (parameters)
         (block (let_declaration (identifier) (deep_ellipsis (integer_literal))))))))
 
+=====================================
+Ellipsis for method / field chains
```

Comment:
There's no ellipsis for method chain here ...
Here is the semgrep.txt for C#:
```
=====================================
Ellipsis in method chain
=====================================
$X = $O.foo(). ... .bar();

---
(compilation_unit
  (global_statement
   (expression_statement
     (assignment_expression
       (identifier)
       (assignment_operator)
       (invocation_expression
         (member_access_expression
           (member_access_ellipsis_expression
              (invocation_expression
                (member_access_expression (identifier) (identifier))
               (argument_list))
              (ellipsis))
          (identifier))
         (argument_list))))))
```

Reply:
🤦‍♂️ I'll make the change to that and update it. I guess I just had "method chain" in my head and forgot the ellipsis part.

Reply:
#310